modules/renderer/map.js

```diff
@@ -311,7 +311,7 @@ export function rendererMap(context) {
     function resetTransform() {
         if (!transformed) return false;
 
-        surface.selectAll('.radial-menu').interrupt().remove();
+        surface.selectAll('.edit-menu').interrupt().remove();
```

Comment:
If we're going to keep `.radial-menu` around for backward compatibility and fork projects, we need to include it in the selector here.